src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -857,11 +861,14 @@ string BuildCommentsString(const SourceLocation& location,
   }
 
   for (int i = 0; i < lines.size(); i++) {
-    // HeaderDoc uses '\' and '@' for markers; escape them.
-    const string line = StripPrefixString(
-        StringReplace(lines[i], "\\", "\\\\", true), " ");
-    final_comments +=
-        prefix + StringReplace(line, "@", "\\@", true) + suffix;
+    string line = StripPrefixString(lines[i], " ");
+    // HeaderDoc and appledoc use '\' and '@' for markers; escape them.
+    line = StringReplace(line, "\\", "\\\\", true);
+    line = StringReplace(line, "@", "\\@", true);
+    // Decouple / from * to not have inline comments inside comments.
+    line = StringReplace(line, "/*", "/\\*", true);
+    line = StringReplace(line, "*/", "*\\/", true);
```

Comment:
Sadly, Xcode shows the escape on these in the quickhelp it shows.  :(    But I guess we just live with it a hope they aren't common.